cfgov/sheerlike/query.py

```diff
@@ -312,9 +312,12 @@ def search(
         else:
             if use_url_arguments:
                 if 'page' in args_flat:
+                    try:
+                        pagenum = int(args_flat['page'])
+                    except ValueError:
+                        pagenum = 1
                     args_flat['from_'] = int(query_dict.get(
-                        'size', '10')) * (int(args_flat['page']) - 1)
-                    pagenum = int(args_flat['page'])
+                        'size', '10')) * (pagenum - 1)
```

Comment:
If there's a ValueError, or `int(args_flat['page'])` is `1`, won't this equate to 0? Is that okay?